src/displayapp/screens/settings/SettingWakeUp.h

```diff
@@ -20,7 +20,7 @@ namespace Pinetime {
       private:
         Controllers::Settings& settingsController;
         uint8_t optionsTotal;
-        lv_obj_t* cbOption[4];
+        lv_obj_t* cbOption[5];
```

Comment:
There are only 4 options.

Reply:
With this one I was just kinda following what was being done elsewhere, as it was set to 4 when there were only three options, and in the shake to wake pr it it also set to 5 when only 1 option was added.

Reply:
I mean...nothing said i checked that 4 was the right or wrong value..I blindly incremented it as I added a new option ;) 
This should be 4, instead of 5.

Reply:
Yep, just tested it. Works fine with 4.